ConfigSpace/hyperparameters.pyx

```diff
@@ -858,6 +882,303 @@ cdef class NormalFloatHyperparameter(FloatHyperparameter):
         return neighbors
 
 
+    def _pdf(self, vector: np.ndarray) -> np.ndarray:
+        mu = self.mu
+        sigma = self.sigma
+        if self.lower == None:
+            raise ValueError('Need upper and lower limits when using user priors.')
+        else:
+            # regardless of whether we have log input or not, this yields the correct answer,
+            # as the data always comes in transformed
```

Comment:
Commenet above seems to conflict the todo note at the top of the class. 

Either:
* remove the TODO if this comment is correct
* check for the `log` bool being set and return a `NotImplementedError`

Reply:
Thanks, great catch. It is indeed correct, and the to-do note will be removed.